packages/web/src/public-site/components/NavBanner.tsx

```diff
@@ -209,7 +222,12 @@ const NavBanner = (props: NavBannerProps) => {
             className={styles.popup}
             dismissOnMouseLeave
           />
-          <div onClick={onClickSignUp} className={styles.signUp}>
+          <div
```

Comment:
oh yeah for this can we do link?

Reply:
This what you had in mind?
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/a299da21-ede1-4328-9ff4-48dae808ba4d)

Reply:
Yeah perfect!!

Reply:
The only weird thing about this is that it technically handles the route push via this method, but replacing it with a link requires an href. They're set to the same route so there's a tiny bit of redundancy there. 
``` 
  const onClickSignUp = handleClickRoute(
    AUDIUS_SIGN_UP_LINK,
    props.setRenderPublicSite
  )
```
I'm not super familiar with this method, but it seems like there are some other things going on there that I shouldn't remove?